src/MassTransit/Logging/StartedActivity.cs

```diff
@@ -28,6 +29,22 @@ public void Update<T>(SendContext<T> context)
                 AddTag(DiagnosticHeaders.Messaging.BodyLength, context.BodyLength.Value.ToString());
         }
 
+        public void RecordException(Exception exception, bool escaped)
+        {
+            var tags = new ActivityTagsCollection
+            {
+                { DiagnosticHeaders.Exceptions.Escaped, escaped },
```

Comment:
Are these based upon a standard convention from OTEL or made up? Link to source?

Reply:
[Based on specs](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/exceptions.md)